<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>811</width>
      <height>491</height>
     </rect>
    </property>
    <widget class="Login" name="LoginPage"/>
    <widget class="ManageControl" name="ManageControlPage"/>
    <widget class="EntryPersonal" name="EntryPersonalPage"/>
    <widget class="QueryModify" name="QueryModifyPage"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuManager_OS">
    <property name="title">
     <string>Manager OS</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuManager_OS"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Login</class>
   <extends>QWidget</extends>
   <header>Login.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ManageControl</class>
   <extends>QWidget</extends>
   <header>ManageControl.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EntryPersonal</class>
   <extends>QWidget</extends>
   <header>EntryPersonal.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QueryModify</class>
   <extends>QWidget</extends>
   <header>QueryModify.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
